### PR TITLE
More metadata

### PR DESCRIPTION
**Pull request by Jeshibu**
Removed the website scraper in favor of an API scraper, which is much faster and more thorough. Also fixed the problem that occurred when the library wasn't embedded in the oculus software install dir. Could use some more tests.

### CODE REVIEW

File: OculusLibrary.Tests/OculusLibrary.Tests.csproj [hunk 1]
```diff
@@ -72,12 +72,21 @@
...
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Asgard%27s Wrath.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
     <None Include="packages.config" />
+    <Content Include="Sprint Vector.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Vader Immortal.json">
```
Comment (mr-ransel) [inline OculusLibrary.Tests/OculusLibrary.Tests.csproj]: Tests currently fail for me, as this isn't checked into `git`

File: OculusLibrary.Tests/OculusLibrary.Tests.csproj [hunk 2]
```diff
@@ -86,7 +95,13 @@
...
   </ItemGroup>
   <ItemGroup>
-    <Content Include="demo1.html">
+    <Content Include="Asgard%27s Wrath.html">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Sprint Vector.html">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Vader Immortal.html">
```
Comment (mr-ransel) [inline OculusLibrary.Tests/OculusLibrary.Tests.csproj]: Also not checked into `git` atm

File: OculusLibrary.Tests/OculusLibrary.Tests.csproj (hunk 1 above)
Comment (Jeshibu) [reply OculusLibrary.Tests/OculusLibrary.Tests.csproj]: Oops. Had that lined up as a future third test case, but then thought 2 of the same test was enough.

**Review by mr-ransel — APPROVED** [review]
Based on the testing we've seen so far, this looks like it's the way to go!

**Pull request merged**